e2e/test/scenarios/models/reproductions.cy.spec.ts

```diff
@@ -421,7 +421,7 @@ describe("issue 39150", { viewportWidth: 1600 }, () => {
   });
 });
 
-describe("issue 41785", () => {
+describe.skip("issue 41785", () => {
```

Comment:
It turned out to be fixed in master (i.e. v51) only. Now we refresh the query results and it became visible.